EREGCSC-178: Revise url pattern declarations in regulations-site

cgodwin1 opened the pull request:
As part of making things readable, we need to know exactly what is being matched with each URL without referring back to placeholder definitions.

Commit:
Replaced placeholders with full regex terms in url definitions

As part of making things readable, we need to know exactly what is
being matched with each URL without referring back to placeholder
definitions.

Commit:
Improved readability of each URL declaration

Moved arguments after URL patterns to new lines. This ensures URL patterns are front-and-center when viewing.

Commit:
Reduce subsequent argument indentation by 1

Pylint seems to complain about this, although I think 2 tabs makes for better reading...

Commit:
Increased maximum line length from 80 to 130

Lint complaining because URLs are too long... our monitors can handle 130 lines!

Commit:
Fixed a couple more whitespace problems

cgodwin1 committed:
Merge branch 'main' into EREGCSC-178

whytheplatypus commented on regulations/urls.py:
```diff
@@ -27,156 +27,153 @@
     # A regulation section with chrome
     # Example: http://.../201-4/2013-10704
-    url(r'^%s/%s$' % (section_pattern, version_pattern),
+    url(r'^(?P<label_id>[\d]+[-][\w]+)/(?P<version>[-\d\w_]+)$',
```
I find this one particularly hard to differentiate from a few of it's peers, like the one from line 105
```
url(r'^(?P<label_id>[\d]+)/(?P<version>[-\d\w_]+)$',
```
I'm not sure what the options are, but in this example the difference between matching `123` vs `123-something` is what would need to stand out.

whytheplatypus replied:
one option might be a middle-ground between this and the original pattern.. Something like
```
url(r'^<label_id>%s/<version>%s$' % (match_section, match_version),
```

whytheplatypus commented on regulations/urls.py:
```diff
@@ -27,156 +27,153 @@
     # A regulation paragraph with chrome
     # Example: http://.../201-2-g/2013-10704
-    url(r'^%s/%s$' % (paragraph_pattern, version_pattern),
+    url(r'^(?P<label_id>[-\w]+)/(?P<version>[-\d\w_]+)$',
```
Kind of related to the above. If there's a way to emphasize the desired match clearly, this pattern should be more concrete.

Commit:
Move regex patterns to reusable constants

Improves readability by explicitly stating what is being matched
against, thereby avoiding the need to interpret regular expressions.

Commit:
Merge branch 'EREGCSC-178' of github.com:cgodwin1/regulations-site into EREGCSC-178

cgodwin1 committed:
Merge branch 'main' into EREGCSC-178
